app/enterprise/2.1.x/doc-updates.md

```diff
@@ -2,23 +2,29 @@
 title: What's New in the Docs
 ---
 
-What's New in the Docs is an overview of recent updates to the Kong Enterprise user documentation, including new features, updates, and fixes. 
+What's New in the Docs is an overview of updates to the Kong Enterprise user documentation, including new features, updates, and fixes for a specific release. 
 
-## New or Updated in Kong Enterprise 2.1.x Docs
+## Kong Enterprise 2.1.3.0 Docs
+Kong Enterprise 2.1.3.0 docs includes 2.1.x(beta) docs. 
```

Comment:
```suggestion
Kong Enterprise 2.1.3.0 docs includes any changes made for the 2.1.0.0 beta docs. 
```
If we're referring to the specific Kong Enterprise release, then maybe we should do the same for beta?